lib/Search/SearchQuery.php

```diff
@@ -164,7 +164,7 @@ public function getFrom()
      */
     public function getSize()
     {
-        return $this->size;
+        return (int)$this->size;
```

Comment:
Unnecessary casting

Reply:
Just following the logic of phpdoc. Also this return value needed as an integer in calculation of pagination.